.circleci/config.yml

```diff
@@ -517,6 +517,9 @@ jobs:
             - run:
                 command: yarn lint
                 name: Lint
+            - run:
+                command: yarn workspace web stylelint
+                name: Stylelint
```

Comment:
Is style-linting not available for native?

Reply:
No idea, at least we don't use it on native.